docs/UserGuide.md

```diff
@@ -91,55 +78,86 @@ Format:
 `list`
 
 > Lists all recipes that are in the storage, in the chronological order they were added.
-> 
+>
 > Depending on the size of the window, the user may add multiple columns to display more recipes.
-> 
+>
 > :bulb: Tip: The `list` command can be used to reset the most recently searched list to the full recipe list after performing a `find` command!
-> 
+>
 > In order to zoom in on a particular recipe in the list, user can double-click on the corresponding recipe
 
 
 ### Finding a recipe by name: `find`
 
-Have a certain recipe at the back of your mind that you want to refer to? `find` helps to save your time scrolling through your whole list of recipes by displaying
+Have a certain recipe at the back of your mind that you want to refer to?
+Or want to look up all recipes associated with a specific tag?
+
+`find` helps to save your time scrolling through your whole list of recipes by displaying
 only those that match any of your specified keywords straight away.
 
 Format:
-`find KEYWORD [KEYWORDS]...`
+`find [PROPERTY] KEYWORD [KEYWORDS]...`
+* :bulb: Tip: Supported properties: `name`, `tag`
 
-> Recipes matching at least one keyword will be returned, e.g. searching `sandwich fries` will return the search results `cheese fries` and `ham sandwich`
+i.e. `find name KEYWORD [KEYWORDS]...`, `find tag KEYWORD [KEYWORDS]...`
+
+> Adding a property behind `find` is optional, and if no property is specified, `find` defaults to filtering by `name`.
+>
+> All keyword queries are case-insensitive. e.g. `chicken` will match `Chicken`
+>
+> Recipes matching at least one keyword will be returned, e.g. searching `sandwich fries` will match recipes named `cheese fries` and `ham sandwich`
 >
 > Recipes are listed in the chronological order that they were added.
-> 
-> Only the name of the recipe is searched
-> 
-> The search is case-insensitive. e.g. chicken will match Chicken
-> 
+>
 > The order of the keywords does not matter. e.g. ham sandwich will match sandwich ham
-> 
+>
 > Only full words will be matched. e.g. chick will **not** match chicken
 
 Example of usage:
-* `find cheese` returns all recipes with the keyword `cheese`
-* `find prawn tofu` returns all recipes with the keyword `prawn` and all recipes with the keyword `tofu`
+* `find cheese` returns all recipes with the keyword `cheese` in their recipe name
+* `find name prawn tofu` returns all recipes with the keyword `prawn` and all recipes with the keyword `tofu` in their names
```

Comment:
can rephrase to "... returns all recipes with the keyword `prawn` AND/OR `tofu` in their name"

Reply:
good phrasing, that was what i intended but i didn't think of that